projects/api/src/category/category.controller.ts

```diff
@@ -0,0 +1,50 @@
+import { Body, Controller, Delete, Get, Param, Patch, Post } from '@nestjs/common';
+import { ApiCreatedResponse, ApiOperation, ApiTags } from '@nestjs/swagger';
+
+import CategoryService from './category.service';
+import CreateCategoryDto from './dto/create-category.dto';
+import UpdateCategoryDto from './dto/update-category.dto';
+import Category from './entities/category.entity';
+
+@Controller('category')
+@ApiTags('Category')
+class CategoryController {
+  constructor(private readonly categoryService: CategoryService) {}
+
+  @Post()
+  @ApiOperation({ summary: 'Create a new category' })
+  @ApiCreatedResponse({ type: Category })
+  async create(createCategoryDto: CreateCategoryDto): Promise<Category> {
+    return this.categoryService.create(createCategoryDto);
+  }
+
+  @Get()
+  @ApiOperation({ summary: 'Get all categories' })
+  @ApiCreatedResponse({ type: [Category], isArray: true })
```

Comment:
If you put that this ApiCreatedResponse isArray, you don't need to put [Category], this going to cause problems in Swagger